cli/command/stack/kubernetes/stack.go

```diff
@@ -36,7 +36,7 @@ func (s *stack) createFileBasedConfigMaps(configMaps corev1.ConfigMapInterface)
 			continue
 		}
 
-		fileName := path.Base(config.File)
+		fileName := filepath.Base(config.File)
```

Comment:
This works fine, this is what the filepath package takes care of, I believe.
I run everything from Powershell and MinGW64 indifferently and filepath has never failed me.

Given
```
package main

import (
	"fmt"
	"path/filepath"
)

func base(p string) {
	fmt.Println(filepath.Base(p), p)
}

func main() {
	base("/usr/local/bla")
	base(`\usr\local\bla`)
	base(`C:\usr\local\bla`)
	base("C:/usr/local/bla")
	base(`bla`)
}
```
Mingw64, Powershell and Cmd all print the same
```
bla /usr/local/bla
bla \usr\local\bla
bla C:\usr\local\bla
bla C:/usr/local/bla
bla bla
```

Reply:
They all printed ```bla``` (first column).
Yes ```C:/Users/me/somefile``` yields ```somefile```.
What config are you referring to?

Reply:
oh lol, I somehow overlooked the first column, and thought only the last example actually returned `bla` 

> What config are you referring to?

sorry, I was typing from my phone so a bit short; Ignore that part, as it's not relevant

Reply:
I *think* `filepath.FromSlash` my be needed here, because `filepath.Base` is looking for `os.PathSeparator`. For instance, what happens if the path is `/usr/local/bla/`?
But I don't have Windows lying around to test this. There should likely be some tests for this.

Reply:
@thaJeztah pointed out that it's looking at `os.IsPathSeparator` which on Windows accepts both `\\` and `/` as a path separator.